src/test/java/com/metamx/emitter/core/HttpEmitterConfigTest.java

```diff
@@ -40,6 +40,7 @@ public void testDefaults()
     Assert.assertEquals(BatchingStrategy.ARRAY, config.getBatchingStrategy());
     Assert.assertEquals(5 * 1024 * 1024, config.getMaxBatchSize());
     Assert.assertEquals(250 * 1024 * 1024, config.getMaxBufferSize());
+    Assert.assertEquals(Long.MAX_VALUE, config.getFlushTimeOut());
```

Comment:
It is `Long.MAX_VALUE` because all other asserts have actual values instead of their DEFAULTS from `HttpEmitterConfig`. I can change if needed but then probably it will make sense to change all asserts?